.github/workflows/version-notes-status-check.yml

```
@@ -0,0 +1,86 @@
+name: ✅Version and Notes Status Check
+
+
+defaults:
+  run:
+    shell: pwsh
+
+
+env:
+  project-name: BranchValidator
+
+
+on:
+  workflow_dispatch:
+  pull_request:
+    branches: master
+    paths:
+      - "**.cs"
+      - "**.csproj"
+
+
+jobs:
+  version_and_notes_status_check:
+    name: Version and Notes Status Check
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+   
+    - name: Get Branch Name
+      id: get-branch-name
+      run: |
+        Write-Host "::group::Environment Variables";
+        Write-Host "GITHUB_REF: ${{ github.ref }}";
+        Write-Host "GITHUB_HEAD_REF: ${{ github.head_ref }}";
+        Write-Host "::endgroup::";
+
+        $branch = "";
+
+        if ("${{ github.ref }}".startsWith("refs/pull/")) {
+          $branch = "${{ github.head_ref }}";
+          Write-Host "Pull Request Run Head Branch: $branch";
+        } else {
+          $branch = "${{ github.ref }}".Replace("refs/heads/", "");
+          Write-Host "Manual Run Branch: $branch";
+        }
+
+        "branch-name=$branch" >> "$env:GITHUB_OUTPUT";
+
+    - name: Test Step Output
+      run: Write-Host "Branch Name Output ${{ steps.get-branch-name.outputs.branch-name }}";
+
+    - name: Get Version (Manual Run)
```

Refactor this name to:

```suggestion
    - name: Get Version
```